contents/docs/feature-flags/creating-feature-flags.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Creating feature flags
+sidebar: Docs
+showTitle: true
+availability:
+    free: none
+    selfServe: full
+    enterprise: full
+---
+
+To create a new feature flag, go to the [feature flags tab](https://app.posthog.com/feature_flags) in the PostHog app, and click on the "New feature flag" button in the top right.
+
+This presents you with a form where you can complete the details of your new flag:
+
+![How to create a feature flag in PostHog](../../images/features/feature-flags/create-new-feature-flag.png)
+
+Here's a breakdown of each field in the form:
+
+### Key
```

Comment:
```suggestion
## Key
```

Reply:
Haha yeah I had this initially, but tbh I thought the text was a bit too big , so opted for ###